src/core/src/sim/mc/moves/mod.rs

```diff
@@ -65,6 +70,76 @@ fn select_molecule(system: &System, moltype: Option<u64>, rng: &mut Box<Rng>) ->
     }
 }
 
+/// This struct keeps track of the number of times a move was called
+/// and how often it was accepted. It also stores the current maximum displacement,
+/// i.e. the magnitude of a particle's translation/rotation or a volume's change in size.
+/// 
+/// We can modify (`update`) the maximum displacement to achieve a target acceptance rate 
+/// of a move based on the running acceptance.
+pub struct Displacement {
+    /// Current maximum displacement
+    delta: f64,
+    /// Current sample range
+    pub delta_range: Range<f64>,
+    /// Number of move calls
+    pub ncalls: u64, 
+    /// Total: number of accepted moves
+    pub naccepted: u64,
+    /// Total: number of attempted moves
+    pub nattempted: u64,
+    /// Target acceptance for the move
+    target_acceptance: f64,
+    /// Call updates with this frequency
+    update_frequency: u64,
+}
+
+impl Displacement {
+    pub fn new(delta: f64, target_acceptance: f64, update_frequency: u64) -> Displacement {
+        Displacement {
+            delta: delta,
+            delta_range: Range::new(-delta, delta),
+            ncalls: 0,
+            naccepted: 0,
+            nattempted: 0,
+            target_acceptance: target_acceptance,
+            update_frequency: update_frequency,
+        }
+    }
+    
+    /// Change `update_frequency` to zero, which will prevent further updates.
```

Comment:
It could be called before adding the move to the Monte-Carlo simulation. After adding it, you are right that there is no more control over the moves. See also how `MonteCarlo::add` send an error if a move is added after the simulation is started.